src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -131,7 +132,12 @@ def morph(config: MorphConfig):
     status = []
     result = MorphStatus([], 0, 0, 0, [])
 
-    ## TODO we might want to put the catalog and schema check here 
+    catalog_name = config.catalog_name
+    schema_name = config.schema_name
+
+    if catalog_name in (None, "transpiler_test") and schema_name in (None, "convertor_test"): 
```

Comment:
we are pushing catalog creation as part of install script, check PR #106

Reply:
@sundarshankar89 should I take this piece out then?

Reply:
We will complete the install script by today, so you can remove this for now. cc: @vijaypavann-db